zeppelin-web-angular/projects/zeppelin-sdk/src/interfaces/message-common.interface.ts

```diff
@@ -118,6 +118,7 @@ export interface ConfigurationsInfo {
     'zeppelin.notebook.collaborative.mode.enable': string;
     'zeppelin.search.use.disk': string;
     'zeppelin.notebook.s3.cannedAcl': string;
+    'zeppelin.notebook.s3.pathStyleAccess': string;
```

Comment:
Is same change needs to be done in the old zeppelin ui module (zeppelin-web) ?

Reply:
I don't think so any changes for old ui is required as it is using rest service to get all configuration keys which is using ZeppelinConfiguration. In that file I have already made required changes.

Reply:
Thanks for clarify, Could you paste the travis CI link ?

Reply:
https://travis-ci.org/github/jenil-shah135/zeppelin/builds/739240932

Reply:
> https://travis-ci.org/github/jenil-shah135/zeppelin/builds/739240932

Thanks, will merge it if no more comment

Reply:
Thanks